terraform/README.md

```diff
@@ -115,11 +123,11 @@ output = json
 
 ```
 [ct-terraform]
-aws_access_key_id = <<access_key_id from bucket_credentials>>
-aws_secret_access_key = <<secret_access_key from bucket_credentials>>
+aws_access_key_id = <<access_key_id from bucket_credentials or cf service-key>>
+aws_secret_access_key = <<secret_access_key from bucket_credentials or cf service-key>>
 ```
 
-3. Copy `bucket` from `bucket_credentials` to the backend block of `<env>/providers.tf`
+3. Copy `bucket` from `terraform` or `cf service-key` output to the backend block of `<env>/providers.tf`
```

Comment:
Hmm - I'm actually not sure I want to say "for new environments" since for instance, you or Davida would need to retrieve those credentials in order to make changes to `stage`.

You need them to run `plan` to verify that your changes do what you expect, even if we are solely relying on github actions to run the `apply` step.